src/main/telemetry/smartport.c

```diff
@@ -348,19 +349,12 @@ void handleSmartPortTelemetry(void)
                     // the MSB of the sent uint32_t helps FrSky keep track
                     // the even/odd bit of our counter helps us keep track
                     if (smartPortIdCnt & 1) {
-                        tmpui = tmpi = GPS_coord[LON];
-                        if (tmpi < 0) {
-                            tmpui = -tmpi;
-                            tmpui |= 0x40000000;
-                        }
-                        tmpui |= 0x80000000;
+                        tmpui = (abs(GPS_coord[LON] / 100) * 6) | 0x80000000;
+                        if (GPS_coord[LON] < 0) tmpui |= 0xC0000000;
```

Comment:
OpenTX uses 0xC0000000, don't really know why. Must say that both work on my testings, I just changed to match OpenTX code.